plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java

```diff
@@ -90,6 +91,12 @@
   private static final Joiner MISSING_GROUP_MEMBER_JOINER = Joiner.on(", "); //$NON-NLS-1$
 
   private static final String ARGUMENTS_SEPARATOR = "="; //$NON-NLS-1$
+  private static final String PIPELINE_TAB_IMAGE = "PIPELINE_TAB_IMAGE";
+  static {
+    DataflowUiPlugin.getDefault().getImageRegistry().put(PIPELINE_TAB_IMAGE,
+        DataflowUiPlugin.imageDescriptorFromPlugin(DataflowUiPlugin.PLUGIN_ID,
+            "icons/Dataflow_16.png"));
+  }
```

Comment:
This doesn't seem like the right place.  This is either be put in `DataflowUiPlugin#initializeImageRegistry()`, or we should created a `SharedImages` type class.